pcc/index.html

```diff
@@ -145,10 +153,21 @@ <h1 class="display-5 fw-bold">PCC</h1>
 
       const formData = new FormData(form);
 
-      const f = formData.get("f");
-      const g = formData.get("g") ? formData.get("g") : f;
+      var insertedF = formData.get("f");
+      var insertedG = formData.get("g") ? formData.get("g") : f;
 
-      const result = PermutationCalculator.composePermutation(g, f);
+      f = insertedF.replace(/[A-Z]/g, replacer);
+      g = insertedG.replace(/[A-Z]/g, replacer);
+
+      var result = PermutationCalculator.composePermutation(g, f);
+      if (charMap.size != 0) {
+        result = JSON.parse(
```

Comment:
Bitte nimm hierfür zwei for schleifen :sweat_smile:

Reply:
Haha big oof. Dachte vielleicht gehts ja an dir vorbei aber nix da

Reply:
nene Freundchen xD
Aber das ist eh nichmal komplizierter einfach
```js
for([x, valX] of result){
for([y, valY] of valX) {
result[x][y] = result[x][y].replace(...)
}
}
```
oder so ähnlich